Makefile

```diff
@@ -303,7 +303,6 @@ GEN_REVISION       := no
 DEP_APPS           := $(shell $(FRAMEWORK_DIR)/scripts/find_dep_apps.py $(APPLICATION_NAME))
 
 ifeq ($(ENABLE_DAGMC), yes)
-  ENABLE_DAGMC     := ON
```

Comment:
I did this so that I could use `ENABLE_DAGMC` to pass "ON" into OpenMC.

Line 8 here: https://github.com/neams-th-coe/cardinal/blob/devel/config/openmc.mk

Do I not need to do that? I'm not sure I ever tried seeing if some other "truthy" value would be equivalent to ON.

Reply:
Oh I didn't realize the variable was again used on line 8. Okay, I can revert this, and use 'ON' instead of 'yes' on line 340

Reply:
I even forgot to change it in unit/Makefile anyway... very sloppy of me.